src/leaflike/db.clj

```diff
@@ -1,6 +1,8 @@
 (ns leaflike.db
   (:require [clojure.java.jdbc :as jdbc]
-            [leaflike.validator :refer [is-valid-bookmark?]])
+            [leaflike.validator :refer [is-valid-bookmark? is-valid-params?]]
+            [honeysql.core :as sql]
+            [honeysql.helpers :refer :all])
```

Comment:
Don't use `:refer :all`
It's always better to have the alias of the namespace appended to the imported vars.